src/core/include/mp-units/ext/prime.h

```diff
@@ -42,6 +42,86 @@ import std;
 
 namespace mp_units::detail {
 
+// (a + b) % n.
+//
+// Precondition: (a < n).
+// Precondition: (b < n).
+// Precondition: (n > 0).
+[[nodiscard]] consteval uint64_t add_mod(uint64_t a, uint64_t b, uint64_t n)
```

Comment:
Usage of `cstdint` without the `std::` namespace prefix will probably fail to compile in C++ modules build.